File: templates/_composition/_shared/Page.Grid.SampleDataService/Services/SampleDataService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the method `GetGridSampleData()` to the SampleDataService of your project.
//**}

namespace Param_ItemNamespace.Services
{
    public static class SampleDataService
    {
//^^
//{[{

        // TODO UWPTemplates: Remove this once your grid page is displaying real data
        public static ObservableCollection<Order> GetGridSampleData()
        {
            return new ObservableCollection<Order>(AllOrders());
        }
//}]}
    }
}

File: templates/_composition/_shared/Page.ImageGalleryVB/Services/SampleDataService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the method `GetSampleModelDataAsync()` to the SampleDataService of your project.
'**}
Namespace Services
    Public Module SampleDataService
        '^^
        '{[{

        ' TODO WTS: Remove this once your image gallery page is displaying real data
        Public Function GetGallerySampleData() As ObservableCollection(Of SampleImage)
            Dim data = New ObservableCollection(Of SampleImage)()
            For i As Integer = 1 To 10
                data.Add(New SampleImage() With {
                    .ID = $"{i}",
                    .Source = $"ms-appx:///Assets/SampleData/SamplePhoto{i}.png",
                    .Name = $"Image sample {i}"
                })
            Next

            Return data
        End Function
        '}]}
    End Module
End Namespace
